handlers.go

```diff
@@ -306,9 +307,9 @@ func (s *Server) ServeAPI(srv string, logging bool, lsteptok string, cachedir st
 	shuttle.POST("/init", s.handleShuttleInit)
 	shuttle.GET("/list", s.handleShuttleList)
 
-	autoretrieve := admin.Group("/autoretrieve")
-	autoretrieve.POST("/init", s.handleAutoretrieveInit)
-	autoretrieve.GET("/list", s.handleAutoretrieveList)
+	ar := admin.Group("/autoretrieve")
+	ar.POST("/init", s.handleAutoretrieveInit)
+	ar.GET("/list", s.handleAutoretrieveList)
 
 	e.POST("/autoretrieve/heartbeat", s.handleAutoretrieveHeartbeat)
```

Comment:
you missed adding this to the `ar` route group

Reply:
`ar` is an alias to `/admin/autoretrieve`, which is different than `/autoretrieve/heartbeat`